src/styles/App.css

```diff
@@ -1,9 +1,9 @@
 .App {
-  width: 87.5rem;
+  /* width: 87.5rem; */
```

Comment:
[OPTIONALK] Please don't leave any uncommented code in your codebase. It is better to git history if you need the specific code written before. In this way, you will keep your codebase neat, organized and professional.